src/Filters/Autofill.php

```diff
@@ -0,0 +1,15 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Orchid\Filters;
+
+trait Autofill
```

Comment:
I’ve tried several times to think of scenarios where this trait might be unnecessary or could potentially break something, but frankly, I couldn't come up with any.

What do you think—if the value is placed directly in the class, could that cause any issues?

Reply:
I think so too. This feature can only break the logic if someone uses this component incorrectly, but I think this cannot happen in a serious production project.

Reply:
Fix pull request. Move code from trait to Filter class and fix Tests